notebooks/maxwell1_fundamentals/InductionRLcircuit.ipynb

```diff
@@ -11,14 +11,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {
     "collapsed": false
    },
    "outputs": [],
    "source": [
     "# IMPORT PACKAGES\n",
-    "import InductionLoop as IND\n",
+    "import em_examples.InductionLoop as IND\n",
```

Comment:
@dccowan: I have updated the import on this example so it imports from em_examples